### ISSUE TITLE
Login button disabled when we use password manager

### ISSUE THREAD

**Issue by pvsmounish** [fossasia/accounts.susi.ai#726]
**Actual Behaviour**

The login button (at https://accounts.susi.ai) is disabled when using password managers to save the password. Even the button is disabled even we copy paste the email and password.

**Expected Behaviour**

The login button should not be disabled.

**Steps to reproduce it**

Just save the password using Google password manager of chrome when you sign up and go to https://accounts.susi.ai.

You can also reproduce it by just copy pasting the email and password without typing anything.

**Screenshots of the issue**

Using a password manager:
![susi accounts login using password manager](https://user-images.githubusercontent.com/16369527/53299244-ac93eb80-385d-11e9-85c8-1a959bcbc932.JPG)

Copying and pasting email and password:
![susi accounts login using password manager](https://user-images.githubusercontent.com/16369527/53299391-3ee8bf00-385f-11e9-96a1-ffd9548d6157.JPG)

**Would you like to work on the issue?**

Yes

**Comment by AakashMallik** [comment]
@pvsmounish Yeah, the logic is a bit messed up. It would be great if you could fix it. :+1:
